graphql/schema/directives.graphql

```diff
@@ -58,3 +58,9 @@ directive @redactSecrets on INPUT_FIELD_DEFINITION | ARGUMENT_DEFINITION | INPUT
 requirePatchOwner is used to restrict certain actions to the patch creator, patch admin, project admin or superuser.
 """
 directive @requirePatchOwner on ARGUMENT_DEFINITION
+
+"""
+requireEVGAdmin is used to restrict certain actions to admins.
+"""
+
+directive @requireEVGAdmin on FIELD_DEFINITION
```

Comment:
maybe `requireAdmin`? Since we're in the Evergreen repo, the mention of `EVG` may be redundant